web/client/configs/localConfig.json

```diff
@@ -41,12 +41,8 @@
       }, {
         "urlPattern": ".*rest/config.*",
         "method": "bearer"
-      },
-      {
-        "urlPattern": "http(s)?\\:\\/\\/gs-stable\\.geo-solutions\\.it(\\:443|\\:80)?\\/geoserver/.*",
-        "authkeyParamName": "authkey",
-        "method": "authkey"
-    }],
+      }
```

Comment:
this must not be commited, eventually you can keep it only locally

Reply:
Fixed it